src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -161,13 +161,14 @@ public Uni<RestResponse<CommandResult>> postCommand(
           @Size(min = 1, max = 48)
           String collection) {
     return schemaCache
-        .getCollectionSettings(
+        .getSchemaObject(
             dataApiRequestInfo, dataApiRequestInfo.getTenantId(), namespace, collection)
         .onItemOrFailure()
         .transformToUni(
-            (collectionProperty, throwable) -> {
+            (schemaObject, throwable) -> {
               if (throwable != null) {
                 Throwable error = throwable;
+                // TODO PUT PROPER {} ON THIS IF STATEMENT
                 if (throwable instanceof RuntimeException && throwable.getCause() != null)
```

Comment:
This actually seems like wrong way around; wonder if fixing order would resolve #1265 (will handle in follow-up)